static/style.css

```
@@ -0,0 +1,157 @@
+.medium-title {
+    font-size: .75em;
+    font-weight: 600;
+}
+
+.small-text {
+    font-size: .5em;
+    font-weight: 400;
+}
+
+/* View transitions */
+@supports (navigation: auto) {
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)